tech/post/2017-11-10-Kubernetes-Affinity&AntiAffinity.md

```diff
@@ -0,0 +1,374 @@
+# Affinity and AntiAffinity
+
+## 1. User Cases
+
+In most cases, Kubernetes will assign the Pods to Nodes by the Scheduler automaticlly.
+But in some special scenarios, the users expect to run some Pods in the particular Nodes.
+For example, the users want to specify some Pods to run in the Nodes which have SSD,
+or specify some kinds of Pods to run in the same Nodes or Zones together.
+Kubernetes provides multiple ways to match these User Cases.
+Affinity and AntiAffinity are that way for the Kubernetes users.

```

Comment:
"Affinity and AntiAffinity are that way for the Kubernetes users" is a little confusing.

Reply:
Done